### PR TITLE
Codes of Lecture 07 corrected

### PR DESCRIPTION
**Pull request by keshashah**
Hi, I am taking the coursera course - HTML, CSS, and Javascript for Web Developers. Found that in Lecture07/
1. The after-ordered-list file was missing
2. The before-ordered-list file had already tags inserted.

Please accept my contribution and merge commit to the codebase - so that it helps other students :)

### CODE REVIEW

File: examples/Lecture07/ordered-lists-before.html [hunk 1]
```diff
@@ -7,22 +7,14 @@
...
+      Open box
+      Take out cookie
+      Make a Double Oreo
+          Peel off the top part
+          Place another cookie in the middle
+          Put back the top part
+      Enjoy!
   </div>
 </body>
-</html>
```
Comment (jasonwr) [inline examples/Lecture07/ordered-lists-before.html]: Remove your newline.

File: examples/Lecture07/ordered-lists-after.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          <li>Put back the top part</li>
+        </ol>
+      </li>
+      <li>Enjoy!</li>
+    </ol>
+
+
+  </div>
+</body>
+</html>
```
Comment (jasonwr) [inline examples/Lecture07/ordered-lists-after.html]: Remove your newline.

Comment (keshashah) [inline examples/Lecture07/ordered-lists-after.html]: Done changes. Please merge it.

File: examples/Lecture07/ordered-lists-before.html (hunk 1 above)
Comment (jasonwr) [inline examples/Lecture07/ordered-lists-before.html]: Still seeing the newline on the line 28. Please remove.

**Comment by ychaikin** [comment]
@keshashah Thank you very much!

**Pull request merged**